modules/standard/BigInteger.chpl

```diff
@@ -3153,7 +3153,18 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
 
 
   // remove
+    /*
+    .. warning::
+
+       bigint.remove is deprecated, use bigint.removeFactor instead
+  */
+  deprecated
+  "bigint.remove is deprecated, use bigint.removeFactor instead"
   proc bigint.remove(const ref a: bigint, const ref f: bigint) : uint {
+    return this.removeFactor(a,f);
+  }
+
+  proc bigint.removeFactor(const ref a: bigint, const ref f: bigint) : uint {
```

Comment:
I might be tempted to rename these arguments to `x` and `fac` respectively while you're here, unless you have better name suggestions.